### DISCUSSION TITLE
Rewrite or masking?

### DISCUSSION THREAD

**Discussion by guicamacho** [vercel/community#5307] [Help]
### Summary

All,

I'm looking to redirect all requests from 

Vercel App #1 to maindomain.com (Main App)
Vercel App #2 to maindomain.com/subfolder (Blog)

Is there a way to get this done which wouldn't impact blog SEO?

### Steps to Reproduce

N/A

**Comment by szmania** [comment]
I'd be curious to know this too.

@guicamacho  check this out: https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain#with-a-proxy-project

I tried the above and I didn't have any luck. I have an issue open for something very similar: https://github.com/orgs/vercel/discussions/5840
